api/constants.go

```diff
@@ -27,6 +28,7 @@ const (
 	RouteAmericas Route = "americas"
 	RouteAsia     Route = "asia"
 	RouteEurope   Route = "europe"
+	RouteSea      Route = "sea"
```

Comment:
```suggestion
	RouteSEA      Route = "sea"
```

SEA is an acronym on this case (South-East Asia).